README.md

```diff
@@ -1,4 +1,4 @@
-# mbpfan
+# mbpfan (With modifications from Rick Sanchez)
```

Comment:
I enjoy Rick and Morty as well but this reference is unnecessary.

Reply:
Thx :)